### ISSUE TITLE
Set up mosaic for blog index?

### ISSUE THREAD

**Issue by bebraw** [antwarjs/default-theme#9]
See bottom of http://www.blurb.com/ . They botched the contrast but the layout looks nice. We can probably implement it without JavaScript even. The newest post can get double space while the rest should layout normally.

The cool thing about this approach would be that it would allow us to benefit from the imagery (header image) attached to each blog post. As we probably don't want to use the images as is at index we probably need to improve our asset pipeline to generate thumbnails automatically.

**Comment by eldh** [comment]
Yep, really like that. We also find a nice solution for bosts that don't have a header image (probably a color based on a hash of the content).

**Comment by bebraw** [comment]
> probably a color based on a hash of the content

The potential problem with that is contrast. Though even that should be solvable if we know the text color beforehand.

Alternatively we can allow the user to set some fallback image, component or such.

**Comment by eldh** [comment]
I think we can assume/enforce a light/white text color.
